Formula/couchdb.rb

```diff
@@ -30,11 +30,18 @@ class Couchdb < Formula
     depends_on "help2man" => :build
   end
 
+  option "with-geocouch", "Build without GeoCouch spatial index extension"
+
   depends_on "spidermonkey"
   depends_on "icu4c"
   depends_on "erlang"
   depends_on "curl" if MacOS.version <= :leopard
 
+  resource "geocouch" do
+    url "https://codeload.github.com/couchbase/geocouch/tar.gz/couchdb1.3.x"
```

Comment:
I still think using the branch and revision is a better idea, personally. It's not a "real" archive tarball as such, and if they push another commit to that branch we'll get a checksum failure again. Just seems like unnecessary future hassle for something we can prevent today.